static/js/tools/timeline/chart_region.tsx

```diff
@@ -233,22 +258,46 @@ class ChartRegion extends Component<ChartRegionPropsType> {
     return result;
   }
 
-  private getSqlQuery(): string {
-    const placeSvPairs = [];
-    for (const place of Object.keys(this.props.placeName)) {
-      for (const sv of Object.keys(this.props.statVarInfo)) {
-        placeSvPairs.push([place, sv]);
+  private shouldShowBqButton(chartGroupInfo: ChartGroupInfo): boolean {
+    for (const mprop of Object.keys(chartGroupInfo.chartIdToStatVars)) {
+      if (!getChartOption(mprop, "delta")) {
+        return true;
       }
     }
-    const placeSvQueryString = placeSvPairs
-      .map(
-        (placeSvPair) =>
-          `(O.observation_about = "${placeSvPair[0]}" AND O.variable_measured = "${placeSvPair[1]}")`
-      )
+    return false;
+  }
+
+  private getQueryForGroup(
```

Comment:
please add a todo here for tests (and there might be opportunity for slight refactoring).

Reply:
I had already put a todo for getSqlQuery() which uses this function